dca/tl_content.php

```diff
@@ -138,8 +139,31 @@
 /**
  * Operations
 **/
-$arrModules = $this->Config->getActiveModules();
-if(!in_array('ce-access',$arrModules))
+$getActiveBundles = function () {
+	$bundles = array_keys(System::getContainer()->getParameter('kernel.bundles'));
+
+	$legacy = [
+		'ContaoCoreBundle'       => 'core',
+		'ContaoCalendarBundle'   => 'calendar',
+		'ContaoCommentsBundle'   => 'comments',
+		'ContaoFaqBundle'        => 'faq',
+		'ContaoListingBundle'    => 'listing',
+		'ContaoNewsBundle'       => 'news',
+		'ContaoNewsletterBundle' => 'newsletter'
+	];
+
+	foreach ($legacy as $bundleName => $module)
+	{
+		if (\in_array($bundleName, $bundles))
+		{
+			$bundles[] = $module;
+		}
+	}
+
+	return $bundles;
+};
+
+if (!in_array('ce-access', $getActiveBundles()))
```

Comment:
ce_access wurde in den Core Integriert, da lohnt sich eine Prüfung nicht mehr ;)

Reply:
Das sagt mir überhaupt nicht :smile: